src/Fabulous.Avalonia/VirtualizedCollection.fs

```diff
@@ -24,7 +24,7 @@ type WidgetDataTemplate(node: IViewNode, templateFn: obj -> Widget) as this =
         let struct (rowNode, view) =
             definition.CreateView(widget, node.TreeContext, ValueSome node)
 
-        let item = ListBoxItem()
+        let item = ContentControl()
```

Comment:
You should be able to not wrap the view at all if you return `view :?> IControl` directly